src/api/ddoc/views.rst

```diff
@@ -315,6 +315,1128 @@ including the update sequence of the database from which the view was
 generated. The returned value can be compared this to the current update
 sequence exposed in the database information (returned by :get:`/{db}`).
 
+Search
```

Comment:
Would a better title be 'Indexes' or 'Search Indexes'?  There are many ways to 'search' in CouchDB, so this seems like an ambiguous title.

Reply:
since this follows the title 'Querying Views and Indexes', I think 'Search' is about as good as you'll get. This is the full-text search feature which _is_ distinct from colloquial uses of the word 'search'.

Reply:
@flimzy also, we have started referring to the set of (Mango, JavaScript views, and Search) all as "secondary indexes", so this is fine.

Reply:
Looking at the rendered layout, though, the parent section is `10.5.5. /db/_design/design-doc/_view/view-name`, which is incorrect for search. Should we move the entire search section one level up so that it's a peer to `_info`, `_view` and `_search_analyze`?

/cc @wohali @flimzy @rnewson 
<img width="304" alt="Screen Shot 2019-06-26 at 9 30 31 PM" src="https://user-images.githubusercontent.com/16679/60226526-a63c6d00-9859-11e9-8917-34635498cd15.png">

Reply:
The challenge is that this is the API reference section - specific to actual endpoints. If this is general search documentation, perhaps it belongs somewhere else, not in the Design Document API reference.

My gut reaction would be to put most of the content in the `ddocs/views` directory as its own top level instead, saving the changes to the API reference explicitly for any endpoint API changes only...though it's worth leaving *something* behind for Search here, since it does indeed end up in a ddoc in the long run.

Also not sure how you're getting 10.x out of this, over on docs.couchdb.org this is [1.5.4](https://docs.couchdb.org/en/latest/api/ddoc/views.html). :confused:

Reply:
Yep OK, that makes sense. But I'd still argue that the Search documentation should go up a level since you hit the `_search` endpoint and not the `_view` one when you make requests.

Reply:
![s4eamxv](https://user-images.githubusercontent.com/112292/60924884-4938b200-a270-11e9-9471-81e18dc0cb0a.jpg)

Reply:
This is done -- well, actually we put the Search User Guide directly under "Design Documents" rather than as a child of "Guide to Views". We also have full documentation for `_search_analyze`, `_search` and `_search_info` in the right places.